src/app/components/OwnerPage/AboutHeading.tsx

```diff
@@ -0,0 +1,16 @@
+
+interface Props {
+  children: React.ReactNode;
+}
+
+const AboutHeading: React.FC<Props> = ({ children }) => {
+  return (
+    <h2 className="text-center font-bold  text-[22px] leading-[33px] tracking-[-0.02em] mb-[40px] mt-[80px] 
+        sm:text-[24px] sm:leading-[36px] 
+        md:text-2xl md:leading-9 md:tracking-tight">
```

Comment:
Here you started well, setting the text size from the smallest breakpoint. Just replace “sm:” with “tablet:”, add “tablet:tracking-[-0.48px]” and remove the styles:
```
md:text-2xl md:leading-9 md:tracking-tight
```
 For tablet and desktop are the same, so there is no need to repeat the code. :)